public/js/src/bridge/bridge_container.tag

```diff
@@ -161,11 +161,6 @@
                     portNums++;
                 }
             });
-            if (portNums > 1 && !alerted.primary) {
```

Comment:
we don't need this check anymore. The logic does not allow for more than one primary port object.